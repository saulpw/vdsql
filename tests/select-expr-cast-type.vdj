#!vd -p
{"longname": "open-file", "input": "tests/business.sqlite", "keystrokes": "o"}
{"sheet": "business", "col": "", "row": "\u30adorder_items", "longname": "open-row", "input": "", "keystrokes": "Enter", "comment": "open current row with sheet-specific dive"}
{"sheet": "order_items", "col": "orderid", "row": "", "longname": "type-int", "input": "", "keystrokes": "#", "comment": "set type of current column to int"}
{"sheet": "order_items", "col": "", "row": "", "longname": "select-expr", "input": "orderid < 30000", "keystrokes": "z|", "comment": "select rows matching Python expression in any visible column"}
{"sheet": "order_items", "longname": "dup-selected"}
