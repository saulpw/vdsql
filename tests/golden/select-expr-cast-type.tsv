orderid	sku	qty	unit_price
23041	JA16	3	2.75
12921	BB93	3	1.83
14345	BB71	4	3.02
14345	BB69	2	3.68
1916	JA16	1	3.00
21167	BB24	2	2.78
19193	JA99	4	2.89
19193	BB69	2	3.75
17098	BB71	4	2.81
2738	BB71	2	2.76
19452	BB71	4	3.24
19452	BB93	4	2.13
3154	JA16	3	3.22
11462	BB71	5	2.70
25822	BB24	5	3.15
25822	BB93	2	2.13
4662	BB71	5	2.91
4662	JA16	5	2.84
12780	BB16	3	2.73
2866	BB93	5	2.10
9471	JA37	5	3.98
9471	BB71	4	3.23
2389	BB16	5	3.01
2389	JA99	2	3.13
6785	BB69	1	3.76
4127	JA90	5	3.23
4127	BB16	1	2.72
6452	BB16	2	3.02
6452	JA99	2	2.98
4410	JA37	5	3.72
4410	JA16	4	2.88
18316	BB71	3	2.89
20066	BB69	4	4.27
6193	JA90	1	3.84
5892	JA90	5	3.53
